src/pages/Merchant/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,88 @@
+import React, { useRef, useState } from "react";
+
+import styles from "./LoginPage.module.scss";
+
+import Button from "components/Button/Button";
+import PasswordInput from "components/InputControl/InputControl";
+import InputControl from "components/InputControl/InputControl";
+
+function LoginPage() {
+  const emailRef = useRef();
+  const passRef = useRef();
+  const [errors, setErrors] = useState({});
+
+  const validateForm = () => {
+    const dummyErrors = {};
+    if (emailRef.current.value === "") {
+      dummyErrors.email = "Enter email";
+      setErrors(dummyErrors);
+    } else {
+      if (
+        !/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(
+          emailRef.current.value
+        )
+      ) {
+        dummyErrors.email = "Enter valid email";
+        setErrors(dummyErrors);
+      }
+    }
+    if (passRef.current.value === "") {
+      dummyErrors.password = "Enter password";
+      setErrors(dummyErrors);
+    }
+  };
+
+  const handleFormSubmit = (event) => {
+    event.preventDefault();
+    validateForm();
```

Comment:
validateFrom() will only update the errors but do not interrupt the execution of this function. take a case where email is invalid ,then this function will show error but also do not stop the further execution of handleFormSubmit function